uber/site_sections/preregistration.py

```diff
@@ -310,16 +328,12 @@ def register_group_member(self, session, group_id, message='', **params):
 
     def group_extra_payment_form(self, session, id):
         attendee = session.attendee(id)
+        cherrypy.session['return_to'] = 'group_members?id={}&message=Extra+payment+undone'.format(attendee.group_id)
         return {
             'attendee': attendee,
-            'charge':   Charge(attendee, description = '{} kicking in extra'.format(attendee.full_name))
+            'charge':   Charge(attendee, description='{} kicking in extra'.format(attendee.full_name))
         }
 
```

Comment:
This got merged with ``undo_attendee_donation`` since there was really no need to have two separate page handlers for almost identical functionality.